coap-gateway/config.yaml

```diff
@@ -52,6 +52,7 @@ clients:
         useSystemCAPool: false
   authorizationServer:
     ownerClaim: sub
+    deviceIdClaim: ""
```

Comment:
I understand but do we want to really JWT claims spread over the whole configuration? I expected that they are near. And still, device id is used for authorization in case mtls+deviceIdClaim

Reply:
> device id is used for authorization in case mtls+deviceIdClaim
yes, but it has nothing to do with the authorization server client. From the configuration perspective, which at the end represents the system, it looks like the grpc client which communicates with the authorization server needs to know the device claim key. That's not the case. It's the authorization part of the CoAP Gateway, API, which verifies it.

Reply:
moved to apis.coap.authorization